eventuate-tram-view-support-rebuild/src/main/java/io/eventuate/tram/viewsupport/rebuild/TopicPartitionOffsetMessageId.java

```diff
@@ -37,4 +39,12 @@ public long getOffset() {
   public void setOffset(long offset) {
     this.offset = offset;
   }
+
+  public String getDatabaseId() {
```

Comment:
`DatabaseId` sounds like it's the id of the database.
It's a DB generated message Id.
however, it's not clear why the current messageID needs to be tracked on separately for each TopicPartition.

Reply:
It is not necessary, but I thought it is more convenient or something, If you don't think so, I can generate single id for all partitions.